generated/nitclk/tox-system_tests.ini

```diff
@@ -50,6 +50,7 @@ deps =
 
 depends =
     nitclk-coverage: py{35,36,37,38}-nitclk-system_tests
+    nitclk-system_tests: py38-nitclk-wheel_dep,
```

Comment:
unrelated

Reply:
This is the fix from above. There is a lot of overhead with separate PRs and waiting for reviews and approval and merging so this branch will work.